src/v2/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertListItem.tsx

```diff
@@ -1,11 +1,17 @@
-import { Box, Text, Flex, Clickable, Spacer } from "@artsy/palette"
+import { Box, Text, Flex, Clickable, Spacer, Pill } from "@artsy/palette"
 import { createFragmentContainer, graphql } from "react-relay"
+import styled from "styled-components"
 import { RouterLink } from "v2/System/Router/RouterLink"
 import { SavedSearchAlertListItem_item } from "v2/__generated__/SavedSearchAlertListItem_item.graphql"
 import { EditAlertEntity } from "../types"
 
 export type SavedSearchAlertListItemVariant = "active" | "inactive"
 
+const AlertPill = styled(Pill)`
+  pointer-events: none;
+  border-color: #707070; /* black60 */
```

Comment:
Used styled components for this one since we don't have a property in pills for changing the border color.

Reply:
@dzucconi - thoughts about adding this to Palette?

Reply:
@dimatretyak also mentioned [this approach](https://github.com/artsy/force/pull/9526#discussion_r817853889), so if we decide not adding this to palette will go ahead with Dima's suggestion

Reply:
Yeah: Either we add this with design guidance as a real variant or state. Or we don't do it.

Reply:
That's now two examples where we're exiting the design system, though. If we're going to be changing the border color of the pill semi-regularly, we should just add a prop over there, right?

Reply:
It should be a real variant, if design really wants this.

Reply:
```
<Pill borderColor="black60" />
```

Reply:
I like this one 👆  and I think that makes sense for cases like this.

Reply:
@gkartalis - here's an example of how we use variants, which could be applied to `Pill`: https://github.com/artsy/palette/blob/main/packages/palette/src/elements/Banner/Banner.tsx#L64-L85

Reply:
Honestly I'm not convinced this isn't a mistake on design's part. This just looks like the normal square variant. Can we confirm with design that this is intended as a new variant?

Reply:
Oh interesting, didn't look in the Pill component and didn't realize we were already using variants there 👍

Reply:
@dzucconi will check with them and will post an update here 🤘

Reply:
👍 

> although it is not that straightforwards that this is it.

What do you mean by this?

Reply:
While having a quick first look in the component, didn't understand that all I had to do is to pass the `active` prop, and I realized that only after going through figma but it's okay I guess :)

Reply:
Ah yeah; sorry. I also didn't put that together. It looks like the docs for this component could use some attention.